### PR TITLE
fix netdata queries

### PR DESCRIPTION
**Pull request by fmessmer**
- [x] add failing netdata queries into error log
- [x] fix failing queries

### CODE REVIEW

**Comment by fmessmer** [comment]
currently the following query fails:
```
[ERROR] [1669388593.760164]: /cpu_monitor_b1_201662_3957855067133744992_b1(query_netdata): NetData request not successful (url: 'http://127.0.0.1:19999/api/v1/data?chart=sensors.coretemp_isa_0000_temperature&format=json&after=-5', status_code 404
```

@pgehring this is something you can find out on Monday

**Review by pgehring — CHANGES_REQUESTED** [review]
failing queries fixed in https://github.com/fmessmer/cob_command_tools/pull/5

File: cob_monitoring/src/cpu_monitor.py [hunk 1]
```diff
@@ -91,15 +92,29 @@ def check_core_temps(self, interval=1):
             # _ vs -
             netdata_module_name_core_temps = ['sensors.coretemp_isa_0000_temperature',
                                               'sensors.coretemp-isa-0000_temperature']
+            error_count = 0
+            netdata_module_name_err = ''
             for name in netdata_module_name_core_temps:
-                netdata_core_temp, error = query_netdata(name, interval)
+                try:
+                    netdata_core_temp, error = query_netdata(name, interval)
```
Comment (fmessmer) [inline cob_monitoring/src/cpu_monitor.py:99]: please apply the try-execpt-logic to all usages of `query_netdata` and `query_netdata_info`

**Review by fmessmer — COMMENTED** [review]
@pgehring can you finalize this?

**Review by pgehring — CHANGES_REQUESTED** [review]
consider review in https://github.com/fmessmer/cob_command_tools/pull/6

**Pull request merged**